proto/parca/query/v1alpha1/query.proto

```diff
@@ -331,6 +334,39 @@ message FlamegraphNodeMeta {
   parca.metastore.v1alpha1.Line line = 4;
 }
 
+// CallgraphNode represents a node in the graph
+message CallgraphNode {
+  // id is the unique id of the node
+  string id = 1;
+
+  // name is the name of the node
+  string name = 2;
```

Comment:
We need to keep track of the cumulative and flat values in each node.

Also, I would not just have a flat `name`. We should have all the metadata associated with this node, just like the flamegraph nodes do (exactly what Monica said).